corehq/blobs/management/commands/export_selected_blobs.py

```diff
@@ -32,31 +36,79 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('path', help='Path to a file with a list of blob meta JSON to export')
+        parser.add_argument('--meta-file-filter', nargs='?',
+                            help='Only export blobs for metadata in files with filenames matching the filter regex.')
         parser.add_argument('--already_exported', dest='already_exported',
                             help='Pass a file with a list of blob names already exported')
+        parser.add_argument('--use-extracted', action='store_true', default=False, dest='use_extracted',
+                            help="Use already extracted dump if it exists.")
+        parser.add_argument('--output-path', help='Directory to write data files to.')
         parser.add_argument('--json-output', action="store_true", help="Produce JSON output for use in tests")
 
     @change_log_level('boto3', logging.WARNING)
     @change_log_level('botocore', logging.WARNING)
     def handle(self, path, **options):
```

Comment:
This is getting to be a pretty big function with multiple logical sections. What do you think of splitting it up into smaller functions?